runelite-api/src/main/java/net/runelite/api/AnimationID.java

```diff
@@ -76,6 +76,7 @@
 	public static final int GEM_CUTTING_EMERALD = 889;
 	public static final int GEM_CUTTING_RUBY = 887;
 	public static final int GEM_CUTTING_DIAMOND = 886;
+	public static final int GEM_CUTTING_AMETHYST =6295;
```

Comment:
place a space after the `=` to stay in line with the formatting of the rest of the file